internal/tiltfile/tiltfile_test.go

```diff
@@ -4029,6 +4029,24 @@ k8s_yaml('foo.yaml')
 		f.assertNextManifest("foo").ImageTargets[0].OverrideArgs)
 }
 
+// See comments on ImageTarget#MaybeIgnoreRegistry()
+func TestCustomBuildNoBuildNoPush(t *testing.T) {
```

Comment:
not apparent what you're testing here b/c "no build no push" is a kinda wibbly translation of "skips local docker and doesn't use expected ref", and also since it's not immediately obvious what the behavior SHOULD be, I'd love it in the test name. How about:
`TestCustomBuildSkipsLocalDockerAndTagPassedIgnoresLocalRegistry` or something? Wordier but much easier to tell what's going on.

Reply:
done!

someday we'll come up with a custom_build v2!!!